itest/universe_test.go

```diff
@@ -263,6 +266,43 @@ func testUniverseFederation(t *harnessTest) {
 	// should also be able to query for stats specifically for the asset.
 	assertUniverseStats(t.t, bob, 1, 0, 1)
 
+	// We should also be able to fetch the new asset from Bob's Universe,
+	// and query for that asset with the compressed script key.
+	firstAssetID := firstAsset[0].AssetGenesis.AssetId
+	firstScriptKey := hex.EncodeToString(firstAsset[0].ScriptKey)
```

Comment:
Maybe move this to the other universe test? Given doesn't have to specifically do with any federation sync operations.